### ISSUE TITLE
Feature Request: Vehicle Actor

### ISSUE THREAD

**Issue by CaptainBrando283** [Muttley/foundryvtt-shadowdark#808]
Could you add a "Vehicle" Actor for Ships, wagons, Boats, etc...Name, HP, gear slots, passengers, Properties, speed, Description?

**Comment by Muttley** [comment]
Could you provide a location in the rules for vehicles so we can accurately represent the core rules?

**Comment by CaptainBrando283** [comment]
Yes, it is in the Cursed Scroll Shadowdark Zine #3, pg 21. Under "Boats".

**Comment by Muttley** [comment]
We should do this as part of #1015
